File: 02. RFID/Film_SparePart_RFID/Library.Reader/Objects/EmailObject.cs
using System;
using System.Collections.Generic;

namespace Library.Reader.Objects
{
    public class EmailObject
    {
        public EmailObject(string company, string mail_to, string mail_cc, string mail_bcc, bool send_flag, List<ReaderEmailObject> reader_object_list)
        {
            Company = company;
            MailTo = mail_to;
            MailCc = mail_cc;
            MailBcc = mail_bcc;
            SendFlag = send_flag;

            for (int i = 0; i <= reader_object_list.Count - 1; i++)
            {
                Body += string.Format("<h3>{3}. [{1}] at {2}</h3>{0}", Environment.NewLine, reader_object_list[i].DeviceName, reader_object_list[i].LocationDesc, i + 1);
                Body += string.Format("<p>Device Name : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list[i].DeviceName);
                Body += string.Format("<p>IP Address : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list[i].IPAddress);
                Body += string.Format("<p>Time Occured : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list[i].TimeOccured.ToString());
                Body += string.Format("<p>Message : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list[i].Message);
                Body += string.Format("{0}{0}", "<br/>");
            }
        }

        public string Company { get; set; }
        public string MailTo { get; set; }
        public string MailCc { get; set; }
        public string MailBcc { get; set; }
        public bool SendFlag { get; set; }
        public string Body { get; set; }
    }
}

File: 02. RFID/Film_SparePart_RFID/Library.Reader/Objects/EmailObject.vb
Namespace Objects

    Public Class EmailObject


        Public Sub New(ByVal company As String, ByVal mail_to As String, ByVal mail_cc As String, ByVal mail_bcc As String, ByVal send_flag As Boolean, ByVal reader_object_list As List(Of ReaderEmailObject))

            Me.Company = company
            Me.MailTo = mail_to
            Me.MailCc = mail_cc
            Me.MailBcc = mail_bcc
            Me.SendFlag = send_flag

            For i As Integer = 0 To reader_object_list.Count - 1

                Me.Body &= String.Format("<h3>{3}. [{1}] at {2}</h3>{0}", Environment.NewLine, reader_object_list(i).DeviceName, reader_object_list(i).LocationDesc, i + 1)
                Me.Body &= String.Format("<p>Device Name : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list(i).DeviceName)
                Me.Body &= String.Format("<p>IP Address : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list(i).IPAddress)
                Me.Body &= String.Format("<p>Time Occured : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list(i).TimeOccured.ToString)
                Me.Body &= String.Format("<p>Message : <strong>{1}</strong></p>{0}", Environment.NewLine, reader_object_list(i).Message)

                Me.Body &= String.Format("{0}{0}", "<br/>")

            Next

        End Sub

        Public Property Company As String

        Public Property MailTo As String

        Public Property MailCc As String

        Public Property MailBcc As String

        Public Property SendFlag As Boolean

        Public Property Body As String

    End Class


End Namespace

